src/components/search/search.component.tsx

```diff
@@ -17,17 +17,17 @@ import DynamicStylingService from "../../services/dynamic-styling.service";
  */
 interface IProps {
   setUserCurrentLocation: () => void;
-  setEndLocation: (poi: POI) => void;
+  setEndLocation: (location: POI | Building) => void;
   setTravelState: (state: TravelState) => void;
   updateSearchResults: (
     userInput: string,
-    setAState: (poi: POI[]) => void,
+    setAState: (locations: (POI | Building)[]) => void,
```

Comment:
I suggest having a more descriptive function name

Reply:
I agree, have a look at updateSearchResults() in the UtilityService class, which I refactored recently to have more descriptive names

Reply:
Done!